modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/js/components/SearchBar.js

```diff
@@ -61,38 +54,48 @@ export default function SearchBar({
 	const alignElementRef = useRef();
 	const dropdownRef = useRef();
 
-	const {resource} = useResource({
-		fetchOptions: {
-			body: suggestionsContributorConfiguration,
-			credentials: 'include',
-			headers: new Headers({
-				'Accept': 'application/json',
-				'Accept-Language': Liferay.ThemeDisplay.getBCP47LanguageId(),
-				'Content-Type': 'application/json',
-				'x-csrf-token': Liferay.authToken,
-			}),
-			method: 'POST',
-		},
-		fetchPolicy: 'cache-first',
-		link: fetchURL.href,
-		onNetworkStatusChange: (status) => {
-			setNetworkState({
-				error: status === 5,
-				loading: status > 1 && status < 4,
-				networkStatus: status,
+	const _handleFetchSuggestions = (value) => {
```

Comment:
Let's reserve the `_handle*` keyword for functions that associate with an `on*` prop to help with identifying which functions directly associate with an event handler.

We can rename this to `_fetchSuggestions`.